### ISSUE TITLE
[macOS] [M1] Crash when going fullscreen while shaders are compiling

### ISSUE THREAD

**Issue by shinra-electric** [cemu-project/Cemu#658]
Issue: 

Cemu will crash after a few seconds every time if you go fullscreen while compiling cached shaders or loading cached Vulkan pipeline. There is no crash if you wait for compiling/loading to complete. 

Hardware: M1 Pro
macOS: Ventura 13.2
Cemu: v2.0-26
Game: BotW

**Comment by shinra-electric** [comment]
I don’t think this is a duplicate of 369. That one is complaining about a corrupt pipeline cache, making subsequent boots fail.

this one is talking about a crash specifically when going into full screen mode while the cache is compiling. Subsequent boots are fine when not in full screen.

**Comment by JJ-Singh** [comment]
@shinra-electric it crashes even when not in fullscreen.

**Comment by shinra-electric** [comment]
Update: This still happens on the 2.6 release

**Comment by shinra-electric** [comment]
Closing, as it seems to have been fixed at some point.
